client/db/bolt/upgrades_test.go

```diff
@@ -0,0 +1,97 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package bolt
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"go.etcd.io/bbolt"
+)
+
+var dbUpgradeTests = [...]struct {
+	name     string
+	upgrade  upgradefunc
+	verify   func(*testing.T, *bbolt.DB)
+	filename string // in testdata directory
+}{
+	{"upgradeFromV0", v1Upgrade, verifyV1Upgrade, "v0.db.gz"},
+}
+
+func TestUpgrades(t *testing.T) {
+	t.Parallel()
```

Comment:
Shouldn't `t.Parallel` go inside of `t.Run`?

Reply:
Looks like the way to cleanup after a set of parallel tests that share a resource (the test directory in this case) is to wrap the group with another `t.Run` (info here: https://blog.golang.org/subtests) like it was when I ported it over. I'm bringing that back as a result.